packager/packager.cc

```diff
@@ -1542,6 +1556,83 @@ unique_ptr<PackageInfoImpl> createAndPopulatePackageInfo(core::GlobalState &gs,
     return info;
 }
 
+// Metadata for Tarjan's algorithm
```

Comment:
If you have a good reference for what you followed to implement this algorithm, it might be nice to link it here if your reference was not Stripe-internal code.